runbotics/runbotics-orchestrator-ui/src/src-landing/components/PostPagination/PostsPagination.tsx

```diff
@@ -0,0 +1,72 @@
+import { FC } from 'react';
+
+import { useRouter } from 'next/router';
+
+import { BlogPost, Page } from '#contentful/common';
+import If from '#src-app/components/utils/If';
+import CardsSection from '#src-landing/views/sections/blog/CardsSection';
+
+import PaginationNavigation from './PaginationNavigation';
+import styles from './PostsPagination.module.scss';
+import SwitchPageButton from './SwitchPageButton';
+
+export interface PostsPaginationProps {
+    posts: BlogPost[];
+    featuredPost?: BlogPost;
+    cardsSectionRef: React.RefObject<HTMLDivElement>;
+    page: Page;
+}
+
+
+const PostsPagination: FC<PostsPaginationProps> = ({ posts, featuredPost, cardsSectionRef, page }) => {
```

Comment:
use `forwardRef` instead

Reply:
pagination is not important in this component's name. Use `PostsSection` and rename current `PostsSection` to e.g. `PostsGrid`

Reply:
done